### ISSUE TITLE
Crash on CCSXML block at LaTeX

### ISSUE THREAD

**Issue by catap** [jgm/pandoc#6093]
I'm using 
```
pandoc 2.9.1.1
Compiled with pandoc-types 1.20, texmath 0.12, skylighting 0.8.3
Default user data directory: /Users/catap/.local/share/pandoc or /Users/catap/.pandoc
Copyright (C) 2006-2019 John MacFarlane
Web:  https://pandoc.org
This is free software; see the source for copying conditions.
There is no warranty, not even for merchantability or fitness
for a particular purpose.
```

When I've tried to covert LaTeX file to plain text that contains CCSXML bloc it crashed as
```
Error at "source" (line 75, column 11):
unexpected _
expecting \end{CCSXML}
  <concept_id>10010405.10010476.10010936.10003590</concept_id>
```

You can create this block at https://dl.acm.org/ccs and it looks like:
```
\begin{CCSXML}
<ccs2012>
   <concept>
       <concept_id>10002944.10011122.10002945</concept_id>
       <concept_desc>General and reference~Surveys and overviews</concept_desc>
       <concept_significance>500</concept_significance>
       </concept>
 </ccs2012>
\end{CCSXML}

\ccsdesc[500]{General and reference~Surveys and overviews}
```

**Comment by jgm** [comment]
Pandoc can't be made to understand every external LaTeX package.
A question has been raised whether we should be more forgiving of loose underscores: See #6034.

**Comment by catap** [comment]
@jgm expected behaviour is just skip this unsupported block and not crash.

**Comment by jgm** [comment]
Looks like the fix for #6034 was not quite complete. I think it should work better now.
